docs/sphinx/install.rst

```diff
@@ -134,15 +182,256 @@ To build the CUDA Quantum Python API from source using pip, run the following co
     cd cuda-quantum && ./scripts/install_prerequisites.sh
     pip install .
 
-For more information about building the entire C++ and Python API's, please see `Building from Source`_.
+For more information about building the entire C++ and Python API from source, we refer to the `CUDA Quantum GitHub repository`_.
 
-Building from Source
-------------------------------
 
-Instructions for building the Python wheels from source are given in the section :ref:`install-python-wheels`.
-For more information about building the entire C++ and Python API from source, we refer to the `CUDA Quantum GitHub repository`_.
+Development with VS Code
+------------------------------------
 
-.. _CUDA Quantum GitHub repository: https://github.com/NVIDIA/cuda-quantum/blob/main/Building.md
+To facilitate application development with, for example, debugging, code completion, hover information, and so on, 
+we recommend using `VS Code <https://code.visualstudio.com/>`_. VS Code provides a seamless
+development experience on all platforms, and is also available without installation via web browser. 
+This sections describes how to connect VS Code to a running container on your machine.
+The section on :ref:`connecting to a remote host <connect-to-remote>` contains information on
+how to set up your development environment when accessing CUDA Quantum on a remote host instead.
+
+.. _docker-in-vscode:
+
+Using a Docker container
+++++++++++++++++++++++++++++++++++++++++
+
+Before connecting VS Code, open a terminal/shell, 
+and start the CUDA Quantum Docker container following the 
+instructions in the :ref:`section above <install-docker-image>`. 
+
+If you have a GitHub or Microsoft account, we recommend that you connect 
+to a CUDA Quantum container using tunnels. To do so, take a look at the 
+section `Developing with Remote Tunnels`_.
```

Comment:
Tunnels are the one thing that works easily and the same for all scenarios; whether you use Docker, Singularity, develop locally, or remotely, and whether you have a local VS Code installation or use the web interface. In fairness, I wrote this when the installation steps were not necessary since I had intended to include them in the container - I think we should be able to do that but to be on the save side I first wanted to check in with legal. Regardless, did you actually try to follow the steps? Because it is super simple, really.

Reply:
> Regardless, did you actually try to follow the steps? Because it is super simple, really.

No, but I will give them a go once the docs preview is complete. I believe that it probably isn't too hard ... it's more that the Dev Containers are even easier in my opinion (definitely less steps). But it sounds like the reason for providing the tunnels recommendation is for user commonality, which I can understand and appreciate.

Reply:
I made some additional updates to also clean up the Next Steps, and revised the guidance for local development with VS Code + Docker.